target/linux/ramips/image/mt7621.mk

```diff
@@ -732,6 +732,17 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a3004ns-dual
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16128k
+  UIMAGE_NAME := a3004nd
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A3004NS-dual
+  DEVICE_PACKAGES := kmod-usb3 kmod-mt76x2
```

Comment:
Please add `kmod-usb-ledtrig-usbport` here.

Reply:
Odd, it disappeared.. Thanks.

Reply:
I don't know what I did but the name just reverted.. I'll just fix them all and re-post. Sorry!

Reply:
Done